prow/genjobs/gen-private-jobs.sh

```diff
@@ -91,15 +91,29 @@ go run ./genjobs \
   --env BAZEL_BUILD_RBE_JOBS=0,GCS_BUILD_BUCKET=istio-private-build,GCS_ARTIFACTS_BUCKET=istio-private-artifacts,DOCKER_REPOSITORY=istio-prow-build/envoy,ENVOY_REPOSITORY=https://github.com/istio-private/envoy,ENVOY_PREFIX=envoy \
   --repo-whitelist proxy
 
+# istio/release-builder test jobs(s) - pre/postsubmit(s)
+go run ./genjobs \
+  "${COMMON_OPTS[@]}" \
+  --branches=release-1.4,master \
+  --job-type presubmit,postsubmit \
+  --repo-whitelist release-builder \
+  --job-blacklist build-release_release-builder_release-1.4_postsubmit,build-release_release-builder_postsubmit,dry-run_release-builder_release-1.4,dry-run_release-builder_release-1.4_postsubmit,dry-run_release-builder,dry-run_release-builder_postsubmit,publish-release_release-builder_release-1.4_postsubmit,publish-release_release-builder_postsubmit
```

Comment:
Should we use a whitelist here? Its not clear to me what is actual remaining?

Reply:
Sure, I can switch this to a whitelist, so that it is more clear.

Reply:
Switched to an explicit whitelist.

Reply:
I also split up `master` and `release-1.4` strictly for readability.